src/app/public/modules/sectioned-form/fixtures/sectioned-form.component.fixture.html

```diff
@@ -1,5 +1,7 @@
 <div id="activeIndexDiv">active index = {{ activeIndexDisplay }}</div>
-<sky-sectioned-form (indexChanged)="updateIndex($event)">
+<sky-sectioned-form
+  [maintainTabContent]="maintainTabContent"
+  (indexChanged)="updateIndex($event)">
```

Comment:
Closing bracket needs to be on a new line.

Reply:
Done